api/api.lesson.js

```diff
@@ -27,6 +27,11 @@ class LessonAPI {
     const { data } = await instanceAxios.post(`/lesson/mark/${id}`, { mark, lessonStepId })
     return data
   }
+  
```

Comment:
Scoretable more related to the course, not lesson. Isn't it?

Reply:
It's a legacy code, idk why it still here. I'll remove it